packages/babel-preset-gatsby/src/index.js

```diff
@@ -2,14 +2,22 @@ const path = require(`path`)
 
 const resolve = m => require.resolve(m)
 
+const cachePath = filePath => {
+  const { GATSBY_CACHE } = process.env
```

Comment:
to be equal to `--cache-dir` and `GATSBY_BUILD_DIR`

```suggestion
  const { GATSBY_CACHE_DIR } = process.env
```